test/gui/shared/scripts/helpers/ConfigHelper.py

```diff
@@ -49,14 +84,12 @@ def get_config_from_env_file(env):
     'lowestSyncTimeout': 1,
     'middlewareUrl': 'http://localhost:3000/',
     'clientLogFile': '-',
-    'clientRootSyncPath': '/tmp/client-bdd/',
-    'tempFolderPath': gettempdir() + '/client-bdd/temp/',
-    'clientConfigDir': os.path.join(
-        get_config_from_env_file("XDG_CONFIG_HOME"), "ownCloud"
-    ),
-    'guiTestReportDir': os.path.abspath('../reports/'),
+    'clientRootSyncPath':  getClientRootPath(),
+    'tempFolderPath':  path.join(getClientRootPath(), 'temp'),
+    'clientConfigDir': getConfigHome(),
+    'guiTestReportDir': path.abspath(path.join('..', 'reports')),
     'ocis': False,
-    'custom_lib': os.path.abspath('../shared/scripts/custom_lib'),
+    'custom_lib': path.abspath(path.join('..', 'shared', 'scripts', 'custom_lib')),
```

Comment:
a more modern approach for paths is 
```
from pathlib import Path
foo = Path("..") / "shared/scrips/custom_lib"
or
foo = Path("../shared/scrips/custom_lib")
```
But actually 90% of windows accepts forwards slashes in paths.